dbt/adapters/databricks/auth.py

```diff
@@ -15,9 +15,11 @@
 
 class token_auth(CredentialsProvider):
     _token: str
+    _host: str
```

Comment:
Why store this on the token?  It's already on the DatabricksCredentials.

Reply:
Seems like I can't get DatabricksCredentials in DatabricksConnectionManager.

I can just use self.credentials_provider in DatabricksConnectionManager, but there are no host in credentials_provider, so I put a host in the token_auth class.

Could you give me a direction how to get DatabricksCredentials in DatabricksConnectionManager?

Reply:
The BaseDatabricksHelper has a copy of DatabricksCredentials.

Reply:
hmm, but that's an instance...let me think.

Reply:
I'm going to pull down a copy of this PR and see if i can figure it out.

Reply:
yes..think I can't get instance in DatabricksConnectionManager...

Thank you very much!

Reply:
Oh, did you already fix this?

Reply:
No, I can't found a way.. so I still use the way put host in token_auth. so that the host can be retrieve from DatabricksConnectionManager.credentials_provider.